### PR TITLE
MULE-13433: Every exported Mule package must contain api (part6)

### PR DESCRIPTION
**Pull request by pablokraan**

### CODE REVIEW

File: core/src/main/java/org/mule/runtime/core/internal/el/mvel/EventVariableResolverFactory.java [hunk 1]
```diff
@@ -48,19 +47,19 @@ public VariableResolver getVariableResolver(String name) {
     if (event != null) {
       if (FLOW.equals(name) && flowName != null) {
         return new MuleImmutableVariableResolver<>(FLOW, (new FlowContext(flowName)), null);
-      } else if (MULE_EVENT_INTERNAL_VARIABLE.equals(name)) {
-        return new MuleImmutableVariableResolver<>(MULE_EVENT_INTERNAL_VARIABLE, event, null);
+      } else if (MVELExpressionLanguageContext.MULE_EVENT_INTERNAL_VARIABLE.equals(name)) {
```
Comment (elrodro83) [inline core/src/main/java/org/mule/runtime/core/internal/el/mvel/EventVariableResolverFactory.java]: review removed static imports

File: core/src/main/java/org/mule/runtime/core/internal/el/mvel/MuleAliasVariableResolver.java [hunk 2]
```diff
@@ -33,7 +30,13 @@ public MuleAliasVariableResolver(MuleAliasVariableResolver aliasVariableResolver
 
   @Override
   public Object getValue() {
-    return executor.execute(expression, context);
+    ClassLoader contextClassLoader = Thread.currentThread().getContextClassLoader();
```
Comment (elrodro83) [inline core/src/main/java/org/mule/runtime/core/internal/el/mvel/MuleAliasVariableResolver.java]: static import

File: core/src/main/java/org/mule/runtime/core/api/el/ExpressionLanguageContext.java [hunk 3]
```diff
@@ -27,6 +27,14 @@
 
   void addAlias(String alias, String expression);
 
+  /**
+   * Adds an alias that requires access to internal Mule API
+   *
+   * @param alias name to be aliased
+   * @param expression expression to replace the alias.
+   */
+  void addInternalAlias(String alias, String expression);
```
Comment (elrodro83) [inline core/src/main/java/org/mule/runtime/core/api/el/ExpressionLanguageContext.java]: who uses this method?

Comment (pablokraan) [reply core/src/main/java/org/mule/runtime/core/api/el/ExpressionLanguageContext.java]: BatchMelExtension

**Pull request merged**